### ISSUE TITLE
Accuracy 

### ISSUE THREAD

**Issue by Dadatata-JZ** [guybuk/ANCOR#4]
Hi there, 

While I am walking through the code, I realize that for computing InfoNCE loss, the contrastive function creates pseudo labels, which is a list with all "0". However, this one is calculated as well when iterating the products of `logits_and_labels` for computing the accuracy in the train function. Should it be ignored when calculating the accuracy? Since all the labels are "0". 

I may understand incorrectly. Thank you in advance and look forward to some helps.

**Comment by Dadatata-JZ** [comment]
Also, additionally, in the training processing, I realize that `val_dataset` and `fg_val_dataset` are essentially doing the same thing?! Can you clarify the necessity of validating twice? 

Thank you in advance.
